src/Network/Email/Email.php

```diff
@@ -1241,6 +1241,16 @@ public static function configTransport($key, $config = null)
     }
 
     /**
+     * Returns an array containing the named transport configurations
+     *
+     * @return array Array of configurations.
+     */
+    public static function configuredTransport()
```

Comment:
I guess my thinking was that this method will always return a list, while configTransport() can take one or many things.